CHANGES.rst

```diff
@@ -20,11 +20,16 @@ Unreleased
     (`#3059`_)
 -   :func:`send_file` supports :class:`~io.BytesIO` partial content.
     (`#2957`_)
+-   Use Werkzeug's JSONMixin and delete Flask's JSONMixin class;
+    Bump minimum dependency versions to the latest stable versions:
+    Werkzeug >= 0.15
+    (`#3125`_)
 
 .. _#2935: https://github.com/pallets/flask/issues/2935
 .. _#2957: https://github.com/pallets/flask/issues/2957
 .. _#2994: https://github.com/pallets/flask/pull/2994
 .. _#3059: https://github.com/pallets/flask/pull/3059
+.. _#3125: https://github.com/pallets/flask/pull/3125
```

Comment:
Should this reference URL `https://github.com/pallets/flask/pull/3181` instead? GitHub redirects the current link to the issue.

Reply:
Oh, I see what's up... the line you added should be to https://github.com/pallets/flask/issues/3125 (not pull/3125). However, I wouldn't worry about it.